certbot/storage.py

```diff
@@ -392,6 +392,26 @@ def __init__(self, config_filename, cli_config, update_symlinks=False):
         self._check_symlinks()
 
     @property
+    def key_path(self):
```

Comment:
Now that we're no longer using a configuration and lineage in places interchangeably, are these properties needed? Do we need/want to be able to access the same attribute of a lineage inmultiple ways?